### ISSUE TITLE
Wishlist: Details pane > Type -- would like more options

### ISSUE THREAD

**Issue by erikaherzog** [WikiEducationFoundation/WikiEduDashboard#1128]
For the Details pane of programs, under the **Type** dropdown menu, would like to have more choices so there can be granularity and customization of event types. 

See [WM NYC Event types](https://en.wikipedia.org/wiki/Wikipedia:Meetup/NYC/Event_archive#Event_types "WM NYC Event types"):
* WikiWednesday
* WikiConference
* Wikipedia Day
* Institutional collection releases
* Photo competitions
* Wiknic
* etc.

Also, where does this information get captured? Is it part of reports?

**Comment by ragesoss** [comment]
The 'type' here is primarily about how the course/program behaves, rather than categorization by type. In particular, different features, labels, and stats calculations apply to different types. (For example, a Visiting Scholarship type program will only count stats for articles that are assigned, so that we can differentiate between articles a visiting scholar is working on related to that position versus other Wikipedia editing that is unrelated. And the Wiki Ed timeline builder only applies to ClassroomProgramCourse type.)

Having another method of categorizing events along the lines you suggest might be a useful addition. That would need to be separate from the 'type' option, I think (which is mainly an admin-facing feature, and which should almost never need to be changed).

**Comment by erikaherzog** [comment]
Thanks @ragesoss for the explanation.

Is this list -- with attendant constraints -- collected anywhere? I would like to use the correct types as they are being utilized. And then be able to apply them correctly for the various events we do.

– Erika

**Comment by ragesoss** [comment]
@erikaherzog Not really, at this point. The two types that are relevant for P&E dashboard are Editathon and BasicCourse. They don't actually function differently from each other at all this point, and the default BasicCourse type is fine to leave for everything.

I anticipate introducing some new types later, especially to support the education program, and possibly adding some type-specific functionality for Editathons at some point, but for now, the main purpose the different types serve is to allow the system to handle the specific types of Wiki Ed courses on our dashboard, and have separate default behavior on P&E dashboard.

Documenting them will be a good thing to do at some point, especially once we add new types that are relevant for P&E.
